CHANGELOG.md

```diff
@@ -76,7 +86,10 @@ Some new algorithms and features are still under active development, and are ava
-- Various documentation, demo and error message fixes and improvements: [\#1141](https://github.com/stellargraph/stellargraph/pull/1141), [\#1219](https://github.com/stellargraph/stellargraph/pull/1219), [\#1246](https://github.com/stellargraph/stellargraph/pull/1246), [\#1260](https://github.com/stellargraph/stellargraph/pull/1260), [\#1266](https://github.com/stellargraph/stellargraph/pull/1266), [\#1361](https://github.com/stellargraph/stellargraph/pull/1361), [\#1362](https://github.com/stellargraph/stellargraph/pull/1362), [\#1385](https://github.com/stellargraph/stellargraph/pull/1385), [\#1386](https://github.com/stellargraph/stellargraph/pull/1386), [\#1363](https://github.com/stellargraph/stellargraph/pull/1363), [\#1376](https://github.com/stellargraph/stellargraph/pull/1376), [\#1405](https://github.com/stellargraph/stellargraph/pull/1405) (thanks @thatlittleboy), [\#1408](https://github.com/stellargraph/stellargraph/pull/1408), [\#1393](https://github.com/stellargraph/stellargraph/pull/1393), [\#1403](https://github.com/stellargraph/stellargraph/pull/1403), [\#1401](https://github.com/stellargraph/stellargraph/pull/1401), [\#1397](https://github.com/stellargraph/stellargraph/pull/1397), [\#1396](https://github.com/stellargraph/stellargraph/pull/1396), [\#1391](https://github.com/stellargraph/stellargraph/pull/1391), [\#1394](https://github.com/stellargraph/stellargraph/pull/1394), [\#1434](https://github.com/stellargraph/stellargraph/pull/1434) (thanks @thatlittleboy), [\#1442](https://github.com/stellargraph/stellargraph/pull/1442), [\#1438](https://github.com/stellargraph/stellargraph/pull/1438) (thanks @thatlittleboy), [\#1413](https://github.com/stellargraph/stellargraph/pull/1413), [\#1450](https://github.com/stellargraph/stellargraph/pull/1450), [\#1440](https://github.com/stellargraph/stellargraph/pull/1440), [\#1453](https://github.com/stellargraph/stellargraph/pull/1453), [\#1447](https://github.com/stellargraph/stellargraph/pull/1447), [\#1467](https://github.com/stellargraph/stellargraph/pull/1467), [\#1465](https://github.com/stellargraph/stellargraph/pull/1465) (thanks @thatlittlboy), [\#1470](https://github.com/stellargraph/stellargraph/pull/1470), [\#1475](https://github.com/stellargraph/stellargraph/pull/1475), [\#1480](https://github.com/stellargraph/stellargraph/pull/1480)
+- `PaddedGraphGenerator.flow` now also accepts a list of `StellarGraph` objects as input [\#1458](https://github.com/stellargraph/stellargraph/pull/1458)
+- Supervised Graph Classification demo now prints progress update messages during training [\#1485](https://github.com/stellargraph/stellargraph/pull/1485)
+- Explicit contributors file has been removed in order to avoid potentially having inconsistent acknowledgement [\#1484](https://github.com/stellargraph/stellargraph/pull/1484). Please refer to the [Github display for contributors](https://github.com/stellargraph/stellargraph/graphs/contributors) instead.
```

Comment:
👍 good idea

We did have inconsistent acknowledgement, so:

```suggestion
- Explicit contributors file has been removed to avoid inconsistent acknowledgement [\#1484](https://github.com/stellargraph/stellargraph/pull/1484). Please refer to the [Github display for contributors](https://github.com/stellargraph/stellargraph/graphs/contributors) instead.
```